maven-plugin-plugin/src/it/mplugin-370-maven-deps-scope-bad/pom.xml

```diff
@@ -38,6 +38,12 @@ under the License.
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-jxr</artifactId>
+      <version>3.1.1</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
There are no assertions for it ... so here is no need

Reply:
There are assertions to NOT have warnings, so to say, that these "excludes" are obeyed (maven-archiver and maven-jxr).

Reply:
Wrong: removed